Dependency issue, latest -sys version not being downloaded

So I have an Cargo.toml file like this

```
glib = "0.7"
gdk = "0.10"
gtk = { version = "0.6", features = ["v3_22"] }
javascriptcore-rs = "0.6.1"
```

This fails to compile

```
 $ cargo run
  Downloaded javascriptcore-rs v0.6.1
  Downloaded javascriptcore-rs-sys v0.1.1
   Compiling javascriptcore-rs v0.6.1
error[E0412]: cannot find type `JSCValue` in this scope
  --> /home/johan/.cargo/registry/src/github.com-1ecc6299db9ec823/javascriptcore-rs-0.6.1/src/lib.rs:22:10
   |
22 |     raw: JSCValue,
   |          ^^^^^^^^ not found in this scope

error[E0412]: cannot find type `JSCValue` in this scope
   --> /home/johan/.cargo/registry/src/github.com-1ecc6299db9ec823/javascriptcore-rs-0.6.1/src/lib.rs:107:27
    |
107 | impl FromGlibPtrNone<*mut JSCValue> for Value {
    |                           ^^^^^^^^ not found in this scope

error[E0412]: cannot find type `JSCValue` in this scope
   --> /home/johan/.cargo/registry/src/github.com-1ecc6299db9ec823/javascriptcore-rs-0.6.1/src/lib.rs:108:40
    |
108 |     unsafe fn from_glib_none(ptr: *mut JSCValue) -> Self {
    |                                        ^^^^^^^^ not found in this scope

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0412`.
error: Could not compile `javascriptcore-rs`.

To learn more, run the command again with --verbose.
```

As you can see in the log it tries to compile javascriptcore-rs 0.6.1 with javascriptcore-rs-sys 0.1.1 which are incompatible. A quick fix was to change my Cargo.toml to specify javascriptcore-rs-sys version to 0.1.2

```
glib = "0.7"
gdk = "0.10"
gtk = { version = "0.6", features = ["v3_22"] }
javascriptcore-rs = "0.6.1"
javascriptcore-rs-sys = "0.1.2"
```

This shouldn't be needed though, javascriptcore-rs needs some kind of fix for this but I'm not sure how.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "javascriptcore-rs"
-version = "0.6.1"
+version = "0.7.0"
 authors = ["The Gtk-rs Project Developers"]
 
 description = "Rust bindings for the javacriptcore library"
@@ -16,4 +16,4 @@ name = "javascriptcore"
 glib = "^0.7.0"
 [dependencies.javascriptcore-rs-sys]
 path = "./javascriptcore-sys"
-version = "0.1.1"
+version = "0.2.0"

--- a/javascriptcore-sys/Cargo.toml
+++ b/javascriptcore-sys/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "javascriptcore-rs-sys"
-version = "0.1.2"
+version = "0.2.0"
 authors = ["The Gtk-rs Project Developers"]
 
 description = "Sys functions for the Rust bindings of the javacriptcore library"

